Makefile

```diff
@@ -125,9 +125,9 @@ dist/docker-%.tar.gz:
 	tar cf - -C build packages/docker/$* | gzip > dist/docker-$*.tar.gz
 
 dist/containerd-%.tar.gz:
-	${MAKE} build/packages/containerd/1.2.6/ubuntu-16.04
-	${MAKE} build/packages/containerd/1.3.3/ubuntu-18.04
-	${MAKE} build/packages/containerd/1.2.13/rhel-7
+	${MAKE} build/packages/containerd/$*/ubuntu-16.04
```

Comment:
This is installing old versions of containerd that happen to have "beta" in their package name.
```
areed@repldev-andrew:~/src/github.com/replicatedhq/kurl/dist$ tar xzvf containerd-beta.tar.gz
packages/containerd/
packages/containerd/beta/
packages/containerd/beta/rhel-7/
packages/containerd/beta/rhel-7/containerd.io-1.2.0-1.2.beta.2.el7.x86_64.rpm
packages/containerd/beta/ubuntu-16.04/
packages/containerd/beta/ubuntu-16.04/partial/
packages/containerd/beta/ubuntu-16.04/containerd.io_1.2.0~beta.2-1_amd64.deb
packages/containerd/beta/ubuntu-18.04/
packages/containerd/beta/ubuntu-18.04/partial/
packages/containerd/beta/ubuntu-18.04/containerd.io_1.2.0~beta.2-1_amd64.deb
```

Reply:
This is from the list-all-packages function, should be updated